### PR TITLE
Clean up Pixie pipeline

### LINKED ISSUE #845: Pixel clustering/cell clustering file cleanup
**Is your feature request related to a problem? Please describe.**
Just some small miscellaneous stuff I noticed when testing the pixel/cell clustering notebooks.
- [X] I think we talked before about removing the prefix from the individual filenames (and only using the prefix for the folder name). They have been removed from most of them, but they still exist on some files: *_channel_norm.feather, *_pixel_norm.feather. Related to this, I don't think the `create_pixel_matrix` function needs the `pixel_cluster_prefix` argument.
- [X] Related to the above point, currently all the individual filenames for cell clustering still include the prefix. Can we remove those?
- [x] Currently, the first 99.9 normalization file is called *_channel_norm.feather and the second 99.9 normalization is called post_rowsum_chan_norm.feather. Can we make these more similar and intuitive - maybe channel_norm.feather and channel_norm_post_preprocessing.feather - or something like that?
- [x] Related to the point above, the two 99.9 normalization files have different formats. The first one (*_channel_norm.feather) has one column for channels, and a second column for norm values, while the second one (post_rowsum_chan_norm.feather) has a different column for each marker - can we make these two files consistent?
- [x] The threshold we use to filter out low expressing pixels is currently called *_pixel_norm.feather. I think this is slightly confusing because we're not really normalizing using this value. We're using it as a threshold. Can we call it 'pixel_threshold.feather' or something like that? Similarly, in this file, the name of the column is "pixel_norm_val" - can we change this to "pixel_threshold_val"?
- [x] Can we change the name of the weights file from "pixel_weights.feather" to "pixel_som_weights.feather"? Same for the cells - can we change to "cell_som_weights.feather"?
- [x] I don't think it's necessary to save the "pixel_som_to_meta.feather" file. That information is already captured in "pixel_channel_avg_som_cluster.csv." Same for *_cell_clust_to_meta.feather.
- [x] Can we change *_cluster_counts_norm.feather to *_cluster_counts_size_normalized.feather?
- [x] Can we save *_weighted_cell_channel.csv as a feather file, just to be consistent? *_cluster_counts.feather and *_cluster_counts_norm.feather have basically the same dimensions.
- [x] It seems to me that *_cell_mat.feather and *_cluster_counts_norm.feather have basically the same info, but *_cell_mat.feather has an additional columns for the clusters, I don't think we need the count info in multiple places - maybe just append cluster info to "cluster_counts_norm.feather"?
- [x] The channel files are named *_cell_som/meta_cluster_channel_avg.csv, while the pixel cluster count file are named *_cell_som/meta_cluster_count_avgs.csv - can we remove the "s" at the end of "avgs" in the second?
- [x] Can we change the names of the notebooks to "2_Pixie_Cluster_Pixels.ipynb" and "3_Pixie_Cluster_Cells.ipynb" to make it clear these are the notebooks related to the paper? That was brought up by the reviewers - that it wasn't clear what part was Pixie.
- [x] Throughout the two notebooks, they reference other notebooks in ark - can we make sure the names of those notebooks are up to date?

### PR DESCRIPTION
**Pull request by alex-l-kong**
**What is the purpose of this PR?**

Closes #845. See issue for all the changes; we want to introduces some new file naming schemes, as well as some basic data reformatting.

**How did you implement your changes**

See issue page.

### CODE REVIEW

**Review by cliu72 — COMMENTED** [review]
Looks great!

Just two more small comment that I think falls under the category of "clean up." In both notebooks, can we move the "Download the Example Dataset" category to before the "set file paths and parameters" section in pixel clustering and "load parameters for cell clustering" in cell clustering? Basically move the "Download the Example Dataset" categories to the beginning. I think this makes it more clear where the paths are coming from, and also makes it clear that people can skip just the first section if they're using their own data.

Also, In the cell clustering notebook, the section numbers start at 1, but in pixel clustering, they start at 0. Can we make that consistent?

**Comment by alex-l-kong** [comment]
Notebook reorganization changes have been addressed.

File: templates/2_Pixie_Cluster_Pixels.ipynb
Comment (cliu72) [inline templates/2_Pixie_Cluster_Pixels.ipynb:38]: <p>Can we move this sentence: "If you would like to test the features in Ark with an example dataset, run the cell below. It will download a dataset consisting of 11 FOVs with 22 channels. You may find more information about the example dataset in the <a href="https://github.com/angelolab/ark-analysis/raw/f2204a3a3358bd87a3c5b3a9be5f78480f714bcb/README.md#example-dataset" target="_blank">README</a>." to below the cell setting the <code>base_dir</code>  (so it's right before the explanation to overwrite_existing)? Can we also add a sentence saying "If you are using your own data, skip the cell below." to the end of that.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: templates/3_Pixie_Cluster_Cells.ipynb
Comment (cliu72) [inline templates/3_Pixie_Cluster_Cells.ipynb:2]: <p>Same comment here - can we move this sentence: "If you would like to test the features in Ark with an example dataset, run the cell below. It will download a dataset consisting of 11 FOVs with 22 channels. You may find more information about the example dataset in the <a href="https://github.com/angelolab/ark-analysis/raw/f2204a3a3358bd87a3c5b3a9be5f78480f714bcb/README.md#example-dataset" target="_blank">README</a>." to below the cell setting the <code>base_dir</code>  (so it's right before the explanation to overwrite_existing)? Can we also add a sentence saying "If you are using your own data, skip the cell below." to the end of that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: templates/2_Pixie_Cluster_Pixels.ipynb
Comment (cliu72) [inline templates/2_Pixie_Cluster_Pixels.ipynb:38]: <p>Okay last thing I promise, can we change this to "Pixie: pixel clustering notebook" and the cell clustering one to "Pixie: cell clustering notebook"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/856/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Comment by alex-l-kong** [comment]
@cliu72 final documentation request has been addressed.

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Will you make sure that all references to notebook 2 are updated with the new name? The flowchart in the readme, any links elsewhere in the repo that point to the old URL, etc

**Comment by alex-l-kong** [comment]
@ngreenwald old notebook names are fixed, example dataset loading functionality has been moved in the notebooks. @srivarra has the flowchart image on the `README.md`, so he opened #869 to address that.

**Review by ngreenwald — APPROVED** [review]
Looks good to me, I'll let candace take a final look through

**Review by cliu72 — APPROVED** [review]
Looks good!

File: ark/utils/example_dataset_test.py
Comment (ngreenwald) [inline ark/utils/example_dataset_test.py:35]: Once this gets changed back we're good to go

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Just need to update huggingface

File: ark/utils/example_dataset_test.py
Comment (alex-l-kong) [reply ark/utils/example_dataset_test.py:35]: @ngreenwald I make sure that happens right before I merge the PR into `main`. So if you approve this PR I can then update HuggingFace, push a final change on this PR with `revision='main'`, then immediately squash and merge the PR into `main` with minimal downtime.

If we're OK with updating HuggingFace right now with the `revision='main'` commit, I can do that as well, then re-request the review.

**Review by ngreenwald — APPROVED** [review]
Sounds good

**Pull request merged**
